src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -231,22 +233,79 @@ public new static RevitDynamoModel Start(IRevitStartConfiguration configuration)
 
         #region trace reconciliation
 
+        /// <summary>
+        /// This method reconciles the current state of the host with the current trace data, in revit's case
+        /// deleting elements which have been orphaned and exist in trace but were not re-created
+        /// </summary>
+        /// <param name="orphanedSerializables"></param>
         public override void PostTraceReconciliation(Dictionary<Guid, List<ISerializable>> orphanedSerializables)
         {
-            var orphanedIds = new List<int>();
+            // because of multiSerialzableIDs some extra logic is 
+            // required to detect if one multiSerializableID is a subset of another, if thats the case we should not delete it.
+
+            //get all the currentTraceData again
+            RuntimeCore runtimeCore = null;
+            if (this.EngineController != null && (this.EngineController.LiveRunnerCore != null))
+                runtimeCore = this.EngineController.LiveRunnerRuntimeCore;
+
+            if (runtimeCore == null)
+                return;
+
+            //list of UUIDs we will remove from the list of orphans
+            var toRemove = new List<String>();
+
+            //foreach orphaned ID check 2 cases:
+            //1. an orphaned MultID is totally subset in one of the latest MultIDs
+            //2. the latest IDS are subset in a MultID in the orphan list
+            //in either of these cases we must remove the IDs from the orphan list so they are not deleted by accident
+            foreach (var orphan in orphanedSerializables)
+            {
+                // Selecting all nodes that are either a DSFunction,
```

Comment:
we can continue here is the orphaned serializable is not an multiserializableID.